contrib/commit-name.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 last_tag=$(git describe --tags --abbrev=0)
```

Comment:
not part of this issue/pr, but the next line really shouldn't say 0.4.0-dev any more. @yuyichao did you not port the improved Compat.jl version back to contrib?